graph-manager-docs/source/managed-federation/overview.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Managed federation overview
+sidebar_title: Overview
+---
+
+Apollo Graph Manager provides free **managed federation** features for data graphs that use [Apollo Federation](https://www.apollographql.com/docs/apollo-server/federation/introduction/).
+
+With managed federation, your [gateway](https://www.apollographql.com/docs/apollo-server/federation/gateway/) is no longer responsible for combining your [implementing service](https://www.apollographql.com/docs/apollo-server/federation/implementing-services/) schemas into a single federated schema. Instead, your implementing services push their schemas to Graph Manager, which combines them. Your gateway then fetches the federated schema and associated configuration from Graph Manager:
```

Comment:
hmm... that's not exactly right. In actuality, your gateway is still responsible for composing the different definitions, but it's not responsible for fetching them from the implementing services. Composition actually happens _twice_ in managed federation. When you publish an implementing service to the registry, AGM will ensure that the service composes with the rest of the graph and will only update the gateway (and the schema registry) if composition succeeds.

Updating the gateway is actually updating the set of service definitions in GCS, which the gateway is still responsible for composing into an overall schema and running, but it isn't doing that by "introspecting" the underlying implementing services. Happy to talk about the reasons for why we chose to build it that way!

Reply:
Thanks for the clarification! I did not realize that composition happens in both spots. Easy enough to fix!